client/src/pages/pot-instance/components/StatsContainer/CompletedStats/StatsCategoryCount.tsx

```diff
@@ -0,0 +1,53 @@
+import { capitalize, countBy } from 'lodash';
+import { FC, useState, useEffect } from 'react';
+import { PotContent } from 'pages/pot-instance/models';
+import styles from '../styles.module.css';
+
+const COUNT_OF_FOOD_EATEN = 'Count of Food Eaten';
+
+interface CategoryCount {
+  meat?: number;
+  vegetable?: number;
+  seafood?: number;
+}
+
```

Comment:
Its hard to enforce this interface since our categories can be dynamic. It would be better to type this below as `Record<string, number>`.
Also works well with your map below since it iterates based on any array size.

Reply:
Omg! I was looking for something like this. Ty! Need to remember it